src/components/pre/pre.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <slot></slot>
+</template>
+
+<script>
+    export default {
+        name: "pre"
+    }
+</script>
+
+<style scoped>
```

Comment:
I've just merged `style-import` branch, can you please add the style dynamically from the papercss repo, like I have done in other components? Also add `lang="css"` please.

Reply:
will do